### PR TITLE
Add script to detect component issues

### PR DESCRIPTION
**Pull request by sergei-maertens**
Probably good to have this script at-hand so we can run it in existing environments.

### CODE REVIEW

File: bin/report_component_problems.py
Comment (sergei-maertens) [inline bin/report_component_problems.py:1]: We might make this part of the upgrade check? Some of the data migrations silently fix issues in #3710

Comment (Viicos) [reply bin/report_component_problems.py:1]: > Some of the data migrations silently fix issues in #3710

What kind of issues?

Comment (Viicos) [inline bin/report_component_problems.py:1]: Is there a reason to have this as a separate `bin` script, and not "built-in" into django as a management command? (would remove the need to setup openforms in `main`)

File: bin/report_component_problems.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                                ]
+                            )
+                            break
+
+    if not problems:
+        print("No problems found.")
+        return True
+
+    print("Found problems in form definition components.")
+    print("")
```
Comment (Viicos) [inline bin/report_component_problems.py:69]:
```suggestion
    print("Found problems in form definition components.", end="\n\n")
```
:shrug:

**Review by Viicos — APPROVED** [review]

File: bin/report_component_problems.py
Comment (sergei-maertens) [reply bin/report_component_problems.py:1]: if someone sets `sizeMobile: "xxs"` for example, the formio columns themselves use size designators for the bootstrap classes, which doesn't translate to our system, so there may be confusion.

We silently correct `size` and `sizeMobile` to their default value if we can't parse them.

Comment (sergei-maertens) [reply bin/report_component_problems.py:1]: yes, makes it easier for Sjoerd to grab it and apply it on older instances that don't have it, while not needing an entry in `INSTALLED_APPS`

File: bin/report_component_problems.py (hunk 1 above)
Comment (Viicos) [reply bin/report_component_problems.py:69]: Yeah I think both are fine!

File: bin/report_component_problems.py
Comment (Viicos) [reply bin/report_component_problems.py:1]: Ah yes, then this is related to my comment on #3710

**Pull request merged**